src/Crossroads.Launcher/Commands/LauncherRootCommand.cs

```diff
@@ -47,12 +47,6 @@ private async Task<int> LauncherApplicationHandler(IHost host, string args)
             return 1;
         }
 
-        private readonly Option argsOption = new Option<string>(new string[] { "--args" }, "Override arguments.")
-        {
-            Argument = new Argument<string>
-            {
-                Arity = ArgumentArity.ExactlyOne
-            }
-        };
+        private readonly Option argsOption = new Option(new[] { "--args" }, "description", argumentType: typeof(string), arity: ArgumentArity.ExactlyOne);
```

Comment:
Is the value for the description correct here? Please check this

Reply:
thanks for the feedback